phaser/phase.py

```diff
@@ -16,10 +156,26 @@ class ReshapePhase:
 
     Note that just dropping or filtering rows one-by-one, or adding or removing columns no matter how much
     other column values are involved, can be done in a regular phase.
-    LMDTODO    https://github.com/rstudio/cheatsheets/blob/main/data-transformation.pdf
     """
 
-class Phase:
+    def __init__(self, name, context=None, error_policy=None):
+        super().__init__(name, context=context, error_policy=error_policy)
+
+    def reshape(self, row_data):
+        raise PhaserException("Subclass ReshapePhase and define what to do in the reshape method")
```

Comment:
Same as above -- should be documented that it expects a list of rows to be returned.